images/win/toolcache-2019.json

```diff
@@ -11,7 +11,7 @@
     "toolcache-pypy-windows-x86": [
```

Comment:
Syntax the name of packages is different between 2019 and 2016 files: @actions/toolcache vs toolcache

Reply:
Thanks, fixed.